src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {person.age !== undefined && <p className="Person__age">{`I am ${person.age}`}</p>}
```

Comment:
You can check just if `person.age` exist:
```suggestion
    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
```